src/games/strategy/engine/framework/startup/ui/editors/MicroWebPosterEditor.java

```diff
@@ -144,12 +148,21 @@ private void initGame() {
     final ArrayList<String> players = new ArrayList<>();
     try {
       final URL url = new URL(hostUrl + "getplayers.php");
-      final BufferedReader in = new BufferedReader(new InputStreamReader(url.openStream()));
```

Comment:
Not all of these reader objects are closed, so we get a deeply nested try-close block below to close each one.